Makefile.build

```diff
@@ -349,7 +359,7 @@ $(COQPP): $(COQPPCMO) coqpp/coqpp_main.ml
 ###########################################################################
 kernel/uint63.ml: kernel/write_uint63.ml kernel/uint63_x86.ml kernel/uint63_amd64.ml
 	$(SHOW)'WRITE $@'
-	$(HIDE)(cd kernel && ocaml unix.cma $(shell basename $<))
+	$(HIDE)cd kernel && ocaml unix.cma $(shell basename $<) > $(shell basename $@)
```

Comment:
Why cd/basename?

Reply:
Indeed, same, I had missed the parts about uint63.

Reply:
No. That would break it.

The `write_uint63` programs looks for original source files in the current work directory.

Reply:
Oh this program is just `cp`
So use `cp` don't reinvent the wheel

Reply:
Why are you guys hijacking this PR? I’ll drop the commit about write-uint63 and you can discuss that in a separate issue.

Reply:
Another option is to pass the directory to look in as an argument; anyways, I am fine with this as-is.

Reply:
> So use `cp` don't reinvent the wheel

The program does some testing that it is better done in OCaml, but indeed has a reimplementation of cp. We should be able to remove this once we are dune only with a conditional `copy_file` rule as Dune exposes `ocaml -config`, but not yet IMO.

Reply:
> Why are you guys hijacking this PR? I’ll drop the commit about write-uint63 and you can discuss that in a separate issue.

We are not hijacking the PR, we are providing (optional as I noted) feedback on the current implementation.

Reply:
> We are not hijacking the PR, we are providing (optional as I noted) feedback on the current implementation.

But I know how it feels as I've been in your position many times, with PRs blocked for a long time for what you call "hickjacking" reviews. [In fact I have a few now]

Just note that it is not the case here as a) I took this PR despite being really a task for kernel maints, b) the tick is still green so that means our request are not blocking.

I would tho fix the commit message, that seems maybe a bit more serious to me.